src/features/components/molecules/image-text-boxes/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { AcuityFeaturesType } from './types'
+import FlexBox from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import useRegion from 'components/hooks/use-region'
+import LinkButton from 'features/components/atoms/link-button'
+
+interface ImageWithParagraphType {
+    contentData: AcuityFeaturesType
+}
+
+const ImageWithParagraph = ({ contentData }: ImageWithParagraphType) => {
+    const { is_eu } = useRegion()
+    const { mobile_flex_direction, data } = contentData
+    return (
+        <FlexBox.Box container="fluid" direction={'col'} mt={'20x'} md={{ mt: '40x' }}>
+            {data.map((feature, index) => {
+                return (
+                    <FlexBox.Box
+                        key={feature?.id}
```

Comment:
If the array is available for mapping, feature.id should be avaible from the start. Do we need to check if feature is not undefined?

Reply:
sure @habib-deriv  i will remove now